cmd/locode/db/csvlocodedb/utils.go

```diff
@@ -0,0 +1,100 @@
+package csvlocodedb
+
+import (
+	"bytes"
+	"compress/gzip"
+	_ "embed"
+	"encoding/csv"
+	"io"
+	"strconv"
+	"sync"
+
+	locodedb "github.com/nspcc-dev/locode-db/cmd/locode/db"
+)
+
+//go:embed data/countries.csv.gz
+var countriesData []byte
+
+//go:embed data/locodes.csv.gz
+var locodesData []byte
+
+var locodeDataOnce sync.Once
+
+func InitLocodeData() {
```

Comment:
Should be performed on the first `Get`.

Reply:
not sure I agree. what is the initialization time? what if an app wants to have init time (any time duration) and then work as fast (and in a similar manner per every call) as possible?

Reply:
It's pretty small (can be benchmarked), we can add this later if needed.

Reply:
> can be benchmarked

lets have this initial time on some initial machine written somewhere (e.g. this thread)